OpenRA.Game/Network/SyncReport.cs

```diff
@@ -77,9 +77,8 @@ void GenerateSyncReport(Report report)
 							NamesValues = DumpSyncTrait(syncHash.Trait)
 						});
 
-			foreach (var e in orderManager.World.Effects)
+			foreach (var sync in orderManager.World.SyncedEffects)
 			{
-				var sync = e as ISync;
 				if (sync != null)
```

Comment:
I am not sure. We don't check for null at earlier stages so I would like to keep it.